ast/capabilities.go

```diff
@@ -63,3 +66,39 @@ func LoadCapabilitiesJSON(r io.Reader) (*Capabilities, error) {
 	var c Capabilities
 	return &c, d.Decode(&c)
 }
+
+// LoadCapabilitiesVersionJSON loads a JSON serialized capabilities structure from the the specific version.
```

Comment:
Typo "the the"

Reply:
eb94d3dddc8fd499dd8201cb042bffec567ef753